src/test/java/org/fundacionjala/at15/katas/pokerhands/mauricio/PokerHandTest.java

```diff
@@ -0,0 +1,20 @@
+package org.fundacionjala.at15.katas.pokerhands.mauricio;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+public class PokerHandTest {
+
+
```

Comment:
Avoid empty lines that do not help to code readability

Reply:
Thank you very much Jose for the recommendations, I will apply them to my code